hrs-portlets-ps-impl/src/main/resources/app-context/psAppContext.xml

```diff
@@ -64,6 +64,24 @@
           </set>
         </entry>
 
+        <entry key="UW_DYN_BN_EBENEFITS">
+          <set>
+            <value>ROLE_VIEW_BENEFITS</value>
+            <!-- effects 
+               Shows message for new benefit enrollment opportunity.
```

Comment:
👍 appreciate continuing with the convention in this file of documenting what the `ROLE_`s do the first time they appear.